jquery.mosaic.js

```diff
@@ -53,6 +56,10 @@
 				});
 		}
 
+		base.$el.on('jqMosaicRefit', function() {
```

Comment:
I'm not sure if I understood this well, but instead of implementing a new refitting trigger this way, I think we should use the methodology this plugin architecture provides for calling methods inside the plugin, which wouldn't need to implement this trigger. Whenever you want to call the fit method of the plugin, you do it this way:

`$('#YourMosaic').Mosaic('fit');`

Did I understood this part of your code correctly? Thanks!

Reply:
Yes, this is much better. Not sure what I was thinking.

Reply:
:) Happens to me also a lot!